notebooks/sam_tests/cmip6_density_transform_pipeline_dev.ipynb

```diff
@@ -8,7 +8,7 @@
     {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        ds = ds.set_coords(["areacello"]) #this keeps area from getting vertically transformed!
</pre><p>This should be done in the <code>match_metrics</code>  call. If not we need to fix that upstream!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/jbusecke/cmip6_omz/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>